templates/single-password.twig

```diff
@@ -4,11 +4,13 @@
 
 	<article id="post-{{ post.ID }}">
 
-		<header class="page-header">
-			<div class="page-header-background"></div>
-		</header>
+		<div class="container narrow-container">
+			<header class="page-header">
+				<div class="page-header-background"></div>
```

Comment:
Do we need to put this empty markup here? :thinking:

Reply:
Removed the container div [there](https://github.com/greenpeace/planet4-master-theme/pull/1630/commits/476299d44b3ff60cc62036694ad564ee93bc286b)